src/poliastro/frames.py

```diff
@@ -220,6 +222,78 @@ class PlutoICRS(_PlanetaryICRS):
     body = Pluto
 
 
+def _make_rotation_matrix_from_reprs(start_representation, end_representation):
+    """
+    Return the matrix for the direct rotation from one representation to a second representation.
+    The representations need not be normalized first.
+    """
+    A = start_representation.to_cartesian()
+    B = end_representation.to_cartesian()
+    rotation_axis = A.cross(B)
+    rotation_angle = -np.arccos(
+        A.dot(B) / (A.norm() * B.norm())
+    )  # negation is required
+
+    # This line works around some input/output quirks of Astropy's rotation_matrix()
+    matrix = np.array(rotation_matrix(rotation_angle, rotation_axis.xyz.value.tolist()))
+    return matrix
+
+
+_EARTH_NORTH_POLE_GCRS = UnitSphericalRepresentation(lon=0 * u.deg, lat=90 * u.deg)
```

Comment:
I have some doubt here! 

Should this be: 

```
_EARTH_NORTH_POLE_GCRS = UnitSphericalRepresentation(lon=0*u.deg, lat=-66.5*u.deg)

_EARTH_DETILT_MATRIX = _make_rotation_matrix_from_reprs(_EARTH_NORTH_POLE_GCRS, CartesianRepresentation(0, 0, 1))

```
This is taking the angle from the axis that's perpendicular to orbit. But I think the first one is correct. Doing this introduces some negetive signs in the matrix.

Reply:
66.5 deg is 90- 23.5 (Tilt of the Earth Axis)